ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/results/TransactionPendingResult.java

```diff
@@ -194,4 +201,19 @@ public String getR() {
   public String getS() {
     return s;
   }
+
+  @JsonGetter(value = "blockHash")
+  public String getBlockHash() {
+    return blockHash;
+  }
+
+  @JsonGetter(value = "blockNumber")
+  public String getBlockNumber() {
+    return blockNumber;
+  }
+
+  @JsonGetter(value = "transactionIndex")
+  public String getTransactionIndex() {
+    return transactionIndex;
+  }
```

Comment:
Instead of returning a final field that only ever takes the value of null we should just return null, get rid of the fields.

Reply:
Done